services/order.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list,
+        username: str,
+        date: str = None
+) -> Order:
+    with transaction.atomic():
+        new_order = Order.objects.create(
+            user=get_user_model().objects.get(username=username)
+        )
+
+        if date:
+            new_order.created_at = date
+            new_order.save()
+
+        for ticket in tickets:
+            Ticket.objects.create(
+                row=ticket["row"],
+                seat=ticket["seat"],
+                movie_session_id=ticket["movie_session"],
+                order=new_order
+            )
+
+        return new_order
```

Comment:
check [README.md](https://github.com/mate-academy/py-tickets-and-orders#readme) about function return

Reply:
you mean that this function and create_user in user.py shouldn't return anything?